script/Makefile

```diff
@@ -430,7 +430,7 @@ check-licenses:
 maven-overlay:
 	@echo "####### Preparing maven dependencies bundle..."
 	mkdir -p build/_maven_overlay
-	./script/maven_overlay.sh -s "$(STAGING_RUNTIME_REPO)" -d "$(CAMEL_K_RUNTIME_DIR)" $(DEFAULT_RUNTIME_VERSION) build/_maven_overlay
+	./script/maven_overlay.sh build/_maven_overlay
```

Comment:
Maybe it misses a `mvn clean package` before calling the script? or alternatively it should be executed in the script.

Reply:
The maven_overlay.sh contains the clean operation.

Reply:
done